File: algorithms/comparethetriplets/compareTheTriplets.py
#!/bin/python3

import sys

def solve(a0, a1, a2, b0, b1, b2):
    # Complete this function
    resultA = 0
    resultB = 0
    arrA = [a0,a1,a2]
    arrB = [b0,b1,b2]
    
    for i in range(0,3):
        if (arrA[i] > arrB[i] & arrA[i] - arrB[i] != 0):
            resultA += 1
        elif (arrB[i] > arrA[i] & arrB[i] - arrA[i] != 0):
            resultB += 1
     
    array = [resultA, resultB]
    return array

a0, a1, a2 = input().strip().split(' ')
a0, a1, a2 = [int(a0), int(a1), int(a2)]
b0, b1, b2 = input().strip().split(' ')
b0, b1, b2 = [int(b0), int(b1), int(b2)]
result = solve(a0, a1, a2, b0, b1, b2)
print (" ".join(map(str, result)))

File: algorithms/comparethetriplets/compareTheTriplets.rb
#!/bin/ruby

def solve(a0, a1, a2, b0, b1, b2)
    # Complete this function
    resultA = 0
    resultB = 0
    arrA = [a0,a1,a2]
    arrB = [b0,b1,b2]
    
    for i in 0..2
        if arrA[i] > arrB[i] and arrA[i] - arrB[i] != 0
            resultA += 1
        elsif arrB[i] > arrA[i] and arrB[i] - arrA[i] != 0
            resultB += 1
        end
    end
     
    array = [resultA, resultB]
    return array
end

a0, a1, a2 = gets.strip.split(' ')
a0 = a0.to_i
a1 = a1.to_i
a2 = a2.to_i
b0, b1, b2 = gets.strip.split(' ')
b0 = b0.to_i
b1 = b1.to_i
b2 = b2.to_i
result = solve(a0, a1, a2, b0, b1, b2)
print result.join(" ")